src/styles/map.scss

```diff
@@ -0,0 +1,57 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```

Comment:
Shouldn't we use the same font that we are using for the website? Is separate font for map necessary?